core/tests/protractor_utils/ProfilePage.js

```diff
@@ -41,14 +41,23 @@ var ProfilePage = function() {
   };
 
   this.expectCurrUserToHaveProfilePhoto = async function() {
+    await waitFor.visibilityOf(
+      currUserProfilePhoto,
+      'Current User Profile Photo Taking too Long To Display');
     expect(await currUserProfilePhoto.isPresent()).toBe(true);
```

Comment:
The waitFor above will cause the test to fail if the profile photo doesn't appear, so this expectation is redundant. Ditto below

Reply:
It looks like you deleted the waitFor. Sorry, this isn't what I meant. I meant that you should delete the `expect(...)` line after the waitFor